### ISSUE TITLE
Various vulnerabilities from low to high severity

### ISSUE THREAD

**Issue by SurealCereal** [OpusCapita/filemanager#283]
I am getting warnings from `npm install` via `npm audit`:  Besides those, there are also deprecation warnings, like `core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.`

`@opuscapita/filemanager-server: ^1.1.1` -  `8 vulnerabilities (3 low, 1 moderate, 4 high)`
`@opuscapita/react-filemanager: ^1.1.1` - `2 high severity vulnerabilities`
`@opuscapita/react-filemanager-connector-node-v1` - `0 vulnerabilities`

In the meantime I am using `npm-force-resolutions` to get around them:
```
  "resolutions": {
    "helmet-csp": "^2.10.0",
    "lodash": "^4.17.15",
    "minimist": "^1.2.5"
  },
```

**Comment by GuyPaddock** [comment]
We're using the following resolutions here:
```
  "resolutions": {
    "helmet-csp": "^2.10.0",
    "lodash": "^4.17.15",
    "minimist": "^1.2.5",
    "node-fetch": "^2.6.1"
  },
```

**Comment by furick1** [comment]
Vulnerable module: | lodash
Introduced through: | @opuscapita/react-filemanager@1.1.4
Fixed in: | 4.17.11
Vulnerable Function: lodash.hasUnicodeWord

Description:
lodash is a modern JavaScript utility library delivering modularity, performance, & extras.

Affected versions of this package are vulnerable to Regular Expression Denial of Service (ReDoS). It parses dates using regex strings, which may cause a slowdown of 2 seconds per 50k characters.

**Comment by estambakio-sc** [comment]
Lodash updated in #311, changes are released in [1.1.7](https://github.com/OpusCapita/filemanager/releases/tag/v1.1.7). Other dependencies mentioned in first comment are not reported by dependabot as vulnerabilities, therefore these are not updated in scope of this issue.
